### PR TITLE
Remove field `lastActivity` from subscription data

### LINKED ISSUE #8338: Remove lastActivity
Search and remove the `lastActivity` property from the codebase.

### PR DESCRIPTION
**Pull request by ggazzo**
@RocketChat/core 

Closes #8338

### CODE REVIEW

File: packages/rocketchat-ui-sidenav/client/roomList.js [hunk 1]
```diff
@@ -19,17 +19,16 @@ Template.roomList.helpers({
...
 		};
-		let sort = { 't': 1, 'name': 1 };
+		const sort = { 't': 1, 'name': 1 };
 		if (this.identifier === 'f') {
 			query.f = favoritesEnabled;
 		} else {
 			let types = [this.identifier];
 			if (this.identifier === 'activity') {
 				types = ['c', 'p', 'd'];
-				sort = { lastActivity : -1, _updatedAt: -1};
```
Comment (sampaiodiego) [inline packages/rocketchat-ui-sidenav/client/roomList.js]: according to [this](https://github.com/RocketChat/Rocket.Chat/commit/52b43d6a1f01a1fb56c830e5cfa0de4eef445460) it should still sort by `_updatedAt`

Comment (engelgabriel) [reply packages/rocketchat-ui-sidenav/client/roomList.js]: I agree with @sampaiodiego

Comment (ggazzo) [reply packages/rocketchat-ui-sidenav/client/roomList.js]: @sampaiodiego @engelgabriel `sort` is not used if the `indentifier === "activity"`

**Comment by ggazzo** [comment]
@engelgabriel could you review migration 101?

**Pull request merged**